android/fakewallet/src/main/java/com/solana/mobilewalletadapter/fakewallet/ui/authorizedapp/AuthorizeDappFragment.kt

```diff
@@ -94,4 +99,22 @@ class AuthorizeDappFragment : Fragment() {
     companion object {
         private val TAG = AuthorizeDappFragment::class.simpleName
     }
+}
+
+fun AppCompatImageView.loadImage(imgUrl: String?) {
+    imgUrl?.let { url ->
+        val imageLoader = if (url.lowercase(Locale.getDefault()).contains("svg")) {
```

Comment:
Instead of using a filename-based heuristic, let's rely on Coil to do image type detection. Let's just always initialize the imageloader with the `SvgDecoder` factory, and defer filetype detection to it.